tsconfig.json

```diff
@@ -9,9 +9,15 @@
   },
   "typedocOptions": {
     "name": "Chart.js",
+    "mode": "file",
+    "target": "ES6",
+    "includeDeclarations": true,
     "excludeExternals": true,
+    "excludeNotExported": true,
```

Comment:
This didn't work for me. I sent a partial fix in https://github.com/chartjs/Chart.js/pull/7121